draft-ietf-quic-manageability.md

```diff
@@ -1055,16 +1055,17 @@ It is expected that any QoS handling in the network, e.g. based on use of
 DiffServ Code Points (DSCPs) {{?RFC2475}} as well as Equal-Cost
 Multi-Path (ECMP) routing, is applied on a per flow-basis (and not per-packet)
```

Comment:
I would suggest making it clear that this is per transport flow, i.e. and not streams or their datagram equivalent that is called datagram flows. So maybe it would be clearer to state this in relation ship to a QUIC connection?

Reply:
Actually connection"lifetime" is too prescriptive... if you really want/need a sender can change the DSCP within a connection, but this really needs to be infrequent to prevent significant disruption to the CC, replay-protection, etc. I'd rather not drill there. I elsewhere added a ref to an RFC that says all this.

Reply:
/flow/ is used rather a lot in this spec. I think it might be clear, or require a change-all to /network flow/ to avoid potential misreading with QUIC datagram flows? Thoughts?

Reply:
To be clear I didn't mean the full life time of the QUIC connection, I do mean use the same DSCP over a consecutive set of packets and one should really not switch back and forth. So I think we are in agreement. 

Considering that the QUIC spec do have recommendations for path changes, one way to deal with this that requires little text is to basically to treat any change of DSCP as a path when it comes to CC and recovery and basically execute Section 9.4 of RFC 9000. However, I do realize that likely should be speced better as you also have to consider what the validation is for a DSCP change and if there is a risk to use DSCP changes for DoS attacks keeping down the rate. So you are likely correct that keeping this simple and only be clearer on that changes should be very infrequent.

Reply:
@gorryfair could you come to a conclusion here so we can merge? thanks!

Reply:
Actually, the resolution to this discussion also looks like it probably belongs in the Applicability draft (PR #383); will merge this and you can continue the discussion over there :)